drinkscript.js

```diff
@@ -1,29 +1,41 @@
 $(document).ready(function() {
+    var searchResults;
```

Comment:
isn't this the same global variable that the meals script uses? i feel like you'll run into conflicts if you both have a global variable named the same thing...

Reply:
oh yea good point, I'll change it